Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -350,6 +352,7 @@ public void RestartRound()
             LobbySong = _robustRandom.Pick(_lobbyMusicCollection.PickFiles).ToString();
             RandomizeLobbyBackground();
             ResettingCleanup();
+            SoundSystem.Play(Filter.Broadcast(), new SoundCollectionSpecifier("RoundEnd").GetSound());
```

Comment:
I wasn't sure on making this a prototype. For round start audio I did because I also made it preset based but wasn't sure.

Reply:
its basically a prototype as-is (i.e. the data is stored in YAML) so I think this is fine. if someone wants a more complex picking algorithm they can do that themselves since I'm not sure what the criteria for it would even be.